README.md

```diff
@@ -59,7 +59,17 @@ the easiest to work with and is likely what most developers will work with.
 
 ### Developing Python
 
-To start developing the Python code, install ERT in editable mode into a virtualenv:
+You might first want to make sure that some system level packages are installed
+before attempting setup:
+
+```
+apt-get install -y python3-pip python3-dev python3-venv \
+  python3-setuptools python3-wheel
+```
```

Comment:
I think you should rather list the packages, or specify that this applies to debian/ubuntu. If you go for the latter, you probably need the same for RH and Mac as well?

Reply:
oof ja very good point. i could setup a virtual machine to test for RH (i suppose?!), but don't know of any way to test for mac. is it enough if i just list the packages and specify that this information concerns a debian-based setup? and leave it up to RH / mac users to do the same for RH / mac, respectively (if it's even necessary)?

Reply:
Why not say that you need

- pip
- python include headers
- venv
- setuptools
- wheel

and leave it as an exercise to the reader?

Reply:
that sounds fair : )